cmd/remediation-aggregator/main.go

```diff
@@ -210,9 +212,14 @@ func createOneResult(crClient *complianceCrClient, owner metav1.Object, labels m
 	fmt.Printf("Creating %s:%s\n", kind.GroupVersionKind().Kind, name)
 
 	err := backoff.Retry(func() error {
-		err := crClient.client.Create(context.TODO(), res)
+		var err error
```

Comment:
similar to the question about creating the directory, wouldn't it be simpler to try creating the object and turn the create into update if create returns that the object already exists?

Reply:
the Get is needed because we need the object metadata (such as the resourceVersion) else the update fails.